eq/config.h

```diff
@@ -429,13 +376,19 @@ class Config : public fabric::Config< Server, Config, Observer, Layout, Canvas,
                                        LB_TIMEOUT_INDEFINITE ) const;
 
     /**
-     * Handle one config event. Thread safe.
+     * Handle the given event. Thread safe.
      *
      * @param command the event command.
      * @return true if the event requires a redraw, false if not.
      * @version 1.5.1
      */
     EQ_API virtual bool handleEvent( EventICommand command );
```

Comment:
Yes, for EVENT_USER or EVENT_OBSERVER_MOTION, which do not have a more specialized handleEvent( type, struct ).